src/api/app/controllers/webui2/package_controller.rb

```diff
@@ -1,6 +1,11 @@
 module Webui2::PackageController
   def webui2_show
+    @tprj, @tpkg = project_and_package_names(@package)
+    @tprj = params[:targetproject] if params[:targetproject] # allow to override by parameter
+    @tpkg = params[:targetpackage] if params[:targetpackage] # allow to override by parameter
```

Comment:
Why? There is no one making use of this feature in our app.

Reply:
Not even while using it from `osc`?

Reply:
Using the webui from osc? Never happens.

Reply:
ok!